--- a/src/Mod/Sketcher/Gui/Resources/Sketcher.qrc
+++ b/src/Mod/Sketcher/Gui/Resources/Sketcher.qrc
@@ -198,12 +198,14 @@
         <file>icons/geometry/Sketcher_CreateTriangle.svg</file>
         <file>icons/geometry/Sketcher_CreateTriangle_Constr.svg</file>
         <file>icons/geometry/Sketcher_Extend.svg</file>
+        <file>icons/geometry/Sketcher_Extend_Constr.svg</file>
         <file>icons/geometry/Sketcher_Projection.svg</file>
         <file>icons/geometry/Sketcher_Projection_Constr.svg</file>
         <file>icons/geometry/Sketcher_Intersection.svg</file>
         <file>icons/geometry/Sketcher_Intersection_Constr.svg</file>
         <file>icons/geometry/Sketcher_Split.svg</file>
         <file>icons/geometry/Sketcher_ToggleConstruction.svg</file>
         <file>icons/geometry/Sketcher_ToggleConstruction_Constr.svg</file>
         <file>icons/geometry/Sketcher_Trimming.svg</file>
+        <file>icons/geometry/Sketcher_Trimming_Constr.svg</file>
     </qresource>
